### ISSUE TITLE
[BUG] all_flags method returning the wrong value.

### ISSUE THREAD

**Issue by lpoulter** [launchdarkly/node-client#53]
If a flag's rollout is set to <100% but the user is given in the Include section of the web application. 

`variation` returns true, 
`all_flags` shows the flag as false for the user. 

Is this expected behaviour? 
I would expect both methods to hold the same value for the flag.

**Comment by drichelson** [comment]
Hi @lpoulter thanks for reaching out. Will you please email support@launchdarkly.com and we can help you out from there?

**Comment by adamlaunchingdarkly** [comment]
Just for reference, this issue can be successfully reproduced with the following:

```
var LaunchDarkly = require('ldclient-node');

var ld_client = LaunchDarkly.init("YOUR_SDK_KEY")

ld_client.once('ready', function() {

var user_with_key_type_int = {"key" : 123};
console.log("INT: "+JSON.stringify(user_with_key_type_int));

var user_with_key_type_string = {"key" : "123"};
console.log("STRING: "+JSON.stringify(user_with_key_type_string));
console.log("===")
console.log("INT all_flags")
ld_client.all_flags(user_with_key_type_int, function(err, flags) {
console.log(flags);
});

console.log("===")
console.log("STRING all_flags")
ld_client.all_flags(user_with_key_type_string, function(err, flags) {
console.log(flags);
});

console.log("===")
console.log("INT variation")
  ld_client.toggle("variation-test", user_with_key_type_int, false, function(err, show_feature) {
    if (show_feature) {
        console.log("TRUE");
    }
    else {
        console.log("FALSE");
    }

  });

console.log("===")
console.log("STRING variation");
  ld_client.toggle("variation-test", user_with_key_type_string, false, function(err, show_feature) {
    if (show_feature) {
        console.log("TRUE");
    }
    else {
        console.log("FALSE");
    }

  });

console.log("===")
console.log("INT all_flags after variation")
ld_client.all_flags(user_with_key_type_int, function(err, flags) {
console.log(flags);
});

console.log("===")
console.log("STRING all_flags after variation")
ld_client.all_flags(user_with_key_type_string, function(err, flags) {
console.log(flags);
});

});
```

**Comment by jkodumal** [comment]
This is fixed in 3.0.8. Thanks for the report!
